backend/src/main/java/com/capstone/moneytree/model/TransactionStatus.java

```diff
@@ -0,0 +1,7 @@
+package com.capstone.moneytree.model;
+
+public enum TransactionStatus {
```

Comment:
This might need to be changed in the future to be more closely related to the api. See https://alpaca.markets/docs/trading-on-alpaca/orders/#order-lifecycle